src/ensembl/src/content/app/entity-viewer/shared/helpers/tests/transcripts-sorter.test.ts

```diff
@@ -24,26 +24,32 @@ import {
 
 import { createTranscript } from 'tests/fixtures/entity-viewer/transcript';
 
+const createTranscriptWithEmptyMetadata = () => {
+  const transcript = createTranscript();
+  transcript.metadata.canonical = transcript.metadata.mane = null;
```

Comment:
Umm, I am not sure I like this function, for several reasons, one of which being that it claims to create a transcript with empty metadata, but only empties two metadata fields.

Why don't we assign nulls to all metadata fields directly in the `createTranscript` function?

Reply:
updated code, please check.